packages/relay/src/lib/clients/mirrorNodeClient.ts

```diff
@@ -361,6 +362,28 @@ export class MirrorNodeClient {
         if (pathLabel == MirrorNodeClient.GET_CONTRACTS_RESULTS_OPCODES) {
           response = await this.web3Client.get<T>(path, axiosRequestConfig);
         } else {
+          // JS supports up to 53 bits integers, once a number with more bits is converted to a js Number type, the initial value is already lost
+          // in order to fix that, we have to use a custom JSON parser that hooks up before
+          // the default axios JSON.parse conversion where the value will be rounded and lost
+          // JSONBigInt reads the string representation of received JSON and formats each number to BigNumber
```

Comment:
I personally find this block is a tiny bit confusing, would you consider update to something like bellow
```suggestion
          // JavaScript supports integers only up to 53 bits. When a number exceeding this limit is converted to a JS Number type, precision is lost due to rounding.
          // To prevent this, `transformResponse` is used to intercept and process the response before Axios’s default JSON.parse conversion.
          // JSONBigInt reads the string representation from the received JSON and converts large numbers into BigNumber objects to maintain accuracy.```

Reply:
Fixed.